### ISSUE TITLE
Error running on FreeBSD: No module named 'control_delcom904x' 

### ISSUE THREAD

**Issue by duncan-bayne** [linville/delcom904x#1]
Hi, first up, thanks for writing this :smile:  I'm using delcom904x to get an old Delcom light I found in my workshop running for the kids.

Unfortunately, it doesn't seem to install / run properly on FreeBSD.  Following the README instructions, I get:

```
$ pip install delcom904x --verbose
Using pip 23.1.2 from /usr/local/lib/python3.9/site-packages/pip (python 3.9)
Defaulting to user installation because normal site-packages is not writeable
Collecting delcom904x
  Using cached delcom904x-0.3.0-py3-none-any.whl (5.2 kB)
Requirement already satisfied: hidapi in /usr/home/duncan/.local/lib/python3.9/site-packages (from delcom904x) (0.14.0)
Requirement already satisfied: setuptools>=19.0 in /usr/home/duncan/.local/lib/python3.9/site-packages (from hidapi->delcom904x) (68.0.0)
Installing collected packages: delcom904x
  changing mode of /home/duncan/.local/bin/control_delcom904x to 755
Successfully installed delcom904x-0.3.0

$ control_delcom904x --green --red --flash --cycleTraceback (most recent call last):
  File "/home/duncan/.local/bin/control_delcom904x", line 5, in <module>
    from control_delcom904x import main
ModuleNotFoundError: No module named 'control_delcom904x'
```

Versions of things:

* FreeBSD 13.2 amd64
* Python 3.9.17
* Cython version 0.29.35

**Comment by linville** [comment]
I can recreate this, the issue isn't specific to FreeBSD but also affects macOS and Linux. As a workaround, I think the light can still be controlled via code; it's just the CLI that is broken right now.

Since I first created this project, I've learned a lot about Python packaging. I'll try to clean things up a bit and push out non-broken release.

**Comment by linville** [comment]
This should be fixed in 0.4.0. Please re-open if you have any further issues.
